app/src/androidTest/java/com/github/fribourgsdp/radio/ui/SongFragmentTest.kt

```diff
@@ -117,5 +118,5 @@ class SongFragmentTest {
         val scenario = launchFragmentInContainer<SongFragment>(bundle)
         Espresso.onView(ViewMatchers.withId(R.id.editTextLyrics))
             .check(ViewAssertions.matches(ViewMatchers.withHint(R.string.add_your_lyrics)))
-    }
+    }*/
```

Comment:
If the automatic fetching of lyrics on Song Fragment opening is still enabled, you can keep these tests I think.

Reply:
Yes indeed I will reput them.